deeposlandia/datasets/dataset.py

```diff
@@ -0,0 +1,161 @@
+"""Encapsulates datasets in a generic Dataset class, and in some more specific
+classes that inherit from it
+
+"""
+
+import os
+import json
+import math
+from multiprocessing import Pool
+
+import cv2
+from PIL import Image
+
+import numpy as np
+
+from deeposlandia import utils
+
```

Comment:
pep8 : 2 blank lines between the imports and the class

Reply:
Fixed.